esphome/components/tuya/tuya.cpp

```diff
@@ -9,7 +10,8 @@ static const char *TAG = "tuya";
 static const int COMMAND_DELAY = 50;
 
 void Tuya::setup() {
-  this->set_interval("heartbeat", 1000, [this] { this->send_empty_command_(TuyaCommandType::HEARTBEAT); });
+  this->send_empty_command_(TuyaCommandType::HEARTBEAT);
+  this->set_interval("heartbeat", 10000, [this] { this->send_empty_command_(TuyaCommandType::HEARTBEAT); });
```

Comment:
The scheduler is set up in a way that it executes the first time now-ish
So this results in 2 heartbeats.

```suggestion
  this->set_interval("heartbeat", 10000, [this] { this->send_empty_command_(TuyaCommandType::HEARTBEAT); });
```

Reply:
Ah, cool. That makes that a bit neater. I didn't even think to check!